cs10/Starve/Adder.java

```diff
@@ -0,0 +1,32 @@
+public class Adder {
+    public boolean[] halfAdder(boolean bitA, boolean bitB) {
+        boolean carry = bitA & bitB;
+        boolean sum = bitA ^ bitB;
+        boolean[] answer = {carry, sum};
+        return answer;
+    }
+
+    public boolean[] fullAdder(boolean bitA, boolean bitB, boolean carry) {
+        boolean[] halfAdder1 = halfAdder(bitA, bitB);
+        boolean[] halfAdder2 = halfAdder(carry, halfAdder1[1]);
+        boolean fullAdderSum = halfAdder2[1];
+        boolean fullAdderCarry = halfAdder1[0] | halfAdder2[0];
+        boolean[] answer = {fullAdderCarry, fullAdderSum};
+        return answer;
+    }
+
+    public boolean[] byteAdder(boolean[] byteA, boolean[] byteB) {
+        boolean carry = false;
+        boolean[] answer = new boolean[9];
+        for (int i = 0; i < answer.length - 1; i++) {
```

Comment:
```suggestion
        int byteLen = 8;
        boolean[] answer = new boolean[byteLen + 1];
        for (int i = 0; i < byteLen; i++) {
```

클린 코드 규칙중에 `원시값을 포장하라!` 라는 말이 있습니다.
숫자 8은 byte 단위의 길이니깐 이라고 추측이 가능한데,
output 인터페이스를 모르는 사람이 처음 숫자 9를 보면 당황할 수도 있을 것 같아요.

 따로 변수명으로 빼면 어떨까 제안해보았습니다.
중요한 의견은 아니고, 그냥 제 취향문제니깐 꼭 따르지 않으셔도 됩니다.